src/clr-icons/test.ts

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Thanks for adding this, I messed up adding the core build. The old test runner was doing a catch all in the project which included icons. I missed adding this for icons when I split the build.